src/autowiring/test/AutoConfigTest.cpp

```diff
@@ -191,6 +202,33 @@ TEST_F(AutoConfigTest, ContextSetAfter) {
   ASSERT_EQ(mcc->b, 10442);
 }
 
+TEST_F(AutoConfigTest, ContextGetSimple) {
+  AutoCurrentContext ctxt;
+  AutoRequired<MyConfigurableClass> mcc;
+
+  mcc->b = 10442;
```

Comment:
`MyConfigurableClass::b` is an `std::atomic`, not an observable.  The system is behaving as designed here.